strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -237,8 +246,10 @@ public URL resolveResource(RepositoryPath repositoryPath)
         Repository repository = path.getFileSystem().getRepository();
         Storage storage = repository.getStorage();
         ArtifactCoordinates c = RepositoryFiles.readCoordinates(path);
-        RepositoryArtifactIdGroupEntry artifactIdGroup = repositoryArtifactIdGroupService.findOne(storage.getId(), repository.getId(), c.getId());
+        Optional<ArtifactIdGroup> artifactIdGroup = artifactIdGroupRepository.findAllArtifactsInGroup(storage.getId(),
+                                                                                                       repository.getId(),
+                                                                                                       c.getId());
```

Comment:
Cosmetic: Indentation fix.
```suggestion
        Optional<ArtifactIdGroup> artifactIdGroup = artifactIdGroupRepository.findAllArtifactsInGroup(storage.getId(),
                                                                                                      repository.getId(),
                                                                                                      c.getId());
```